android/sdl_android/src/main/java/com/smartdevicelink/util/SdlAppInfo.java

```diff
@@ -181,6 +181,40 @@ public String toString() {
         }
         return vehicleMakesList;
     }
+
+    static boolean checkIfVehicleSupported(List<VehicleType> supportedVehicleList, VehicleType connectedVehicle){
+        if (null == supportedVehicleList || null == connectedVehicle){
+            return false;
+        }
+        if (supportedVehicleList.contains(connectedVehicle)){
+            return true;
+        }
+        for (VehicleType supportedVehicle :supportedVehicleList) {
+            String svMake = supportedVehicle.getMake();
+            String cvMake = connectedVehicle.getMake();
+            if(svMake != null && cvMake != null && cvMake.equalsIgnoreCase(svMake)) {
+                String svModel = supportedVehicle.getModel();
```

Comment:
@kostyaBoss On line 196 I would rename this variable to `supportedVehicleModel`.

Reply:
@kostyaBoss On line 196 you can rename `svModel` to `supportedVehicleModel`.